py/makeqstrdata.py

```diff
@@ -259,8 +259,6 @@ def compress(encoding_table, decompressed, encoded_length_bits, len_translation_
                 current_bit -= 1
     if current_bit != 7:
         current_byte += 1
-    if current_byte > len(decompressed):
-        print("Note: compression increased length", repr(decompressed), len(decompressed), current_byte, file=sys.stderr)
```

Comment:
This computation was wrong, or did you fix it? I wonder if it's just worth commenting this out, so it could be revived for compression testing later.

Reply:
I think the computation is misleading.  See the individual commit message for more info on my findings.

Reply:
Thanks. You write a lot of interesting stuff in commit messages, but I never see that, unless I happen to click on the individual commit. I think that's a deficiency of GitHub that it doesn't make these visible or even give a hint of their existence (or am I missing something). I have tended to write notes like that in the PR instead of in the commit. Where I used to work we didn't use GitHub, so I wrote them in the commit. What do you think? Maybe copy them into the original PR comment?